<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>936</width>
    <height>676</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>936</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>936</width>
     <height>606</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(34, 0, 75);</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item alignment="Qt::AlignLeft">
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>936</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>70</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="minimumSize">
          <size>
           <width>936</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{
	font: 18pt &quot;Century Gothic&quot;;
	color: rgb(255,255,255)
}</string>
         </property>
         <property name="text">
          <string>1. Escoge dos habilidades críticas para la posición</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QFrame" name="frame_2">
      <property name="minimumSize">
       <size>
        <width>936</width>
        <height>616</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QFrame" name="frame_3">
         <property name="minimumSize">
          <size>
           <width>468</width>
           <height>600</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QComboBox" name="skill1_selection">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>90</y>
            <width>211</width>
            <height>31</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QComboBox {
	color: rgb(255, 255, 255);
	background-color: #7900E6;
	border-radius: 10px;
	border: 1px solid #FFFFFF;
	font: 12pt &quot;Century Gothic&quot;;
}

QListView{
	border: 1px solid #FFFFFF;
	background-color: #7900E6;
	border-radius: 10px;
}</string>
          </property>
         </widget>
         <widget class="QLabel" name="skill1_description_title">
          <property name="geometry">
           <rect>
            <x>300</x>
            <y>40</y>
            <width>121</width>
            <height>20</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {
	color: #FFFFFF;
	font: 12pt &quot;Century Gothic&quot;;
}</string>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Descripción&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
         <widget class="QTextBrowser" name="skill1_description">
          <property name="geometry">
           <rect>
            <x>280</x>
            <y>80</y>
            <width>151</width>
            <height>81</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QTextBrowser{
	color: #FFFFFF;
	font: 12pt &quot;Century Gothic&quot;;
	border-radius: 1px;
	border: 1px solid #FFFFFF;
}</string>
          </property>
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Century Gothic'; font-size:12pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
         <widget class="QLabel" name="skill1_title">
          <property name="geometry">
           <rect>
            <x>70</x>
            <y>30</y>
            <width>151</width>
            <height>20</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {
	color: #FFFFFF;
	font: 16pt &quot;Century Gothic&quot;;
	width: 2px;
}</string>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Habilidad 1&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
         <widget class="QTextBrowser" name="skill1_traits">
          <property name="geometry">
           <rect>
            <x>280</x>
            <y>220</y>
            <width>151</width>
            <height>81</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QTextBrowser{
	color: #FFFFFF;
	font: 12pt &quot;Century Gothic&quot;;
	border-radius: 1px;
	border: 1px solid #FFFFFF;
}</string>
          </property>
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Century Gothic'; font-size:12pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
         <widget class="QLabel" name="skill1_traits_title">
          <property name="geometry">
           <rect>
            <x>300</x>
            <y>180</y>
            <width>121</width>
            <height>20</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {
	color: #FFFFFF;
	font: 12pt &quot;Century Gothic&quot;;
}</string>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Habilidades&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
         <widget class="QComboBox" name="skill2_selection">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>370</y>
            <width>211</width>
            <height>31</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QComboBox {
	color: rgb(255, 255, 255);
	background-color: #7900E6;
	border-radius: 10px;
	border: 1px solid #FFFFFF;
	font: 12pt &quot;Century Gothic&quot;;
}

QListView{
	border: 1px solid #FFFFFF;
	background-color: #7900E6;
	border-radius: 10px;
}</string>
          </property>
         </widget>
         <widget class="QTextBrowser" name="skill2_description">
          <property name="geometry">
           <rect>
            <x>270</x>
            <y>360</y>
            <width>151</width>
            <height>81</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QTextBrowser{
	color: #FFFFFF;
	font: 12pt &quot;Century Gothic&quot;;
	border-radius: 1px;
	border: 1px solid #FFFFFF;
}</string>
          </property>
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Century Gothic'; font-size:12pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
         <widget class="QLabel" name="skill2_description_title">
          <property name="geometry">
           <rect>
            <x>290</x>
            <y>320</y>
            <width>121</width>
            <height>20</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {
	color: #FFFFFF;
	font: 12pt &quot;Century Gothic&quot;;
}</string>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Habilidad 2&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
         <widget class="QTextBrowser" name="skill2_traits">
          <property name="geometry">
           <rect>
            <x>270</x>
            <y>500</y>
            <width>151</width>
            <height>81</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QTextBrowser{
	color: #FFFFFF;
	font: 12pt &quot;Century Gothic&quot;;
	border-radius: 1px;
	border: 1px solid #FFFFFF;
}</string>
          </property>
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Century Gothic'; font-size:12pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
         <widget class="QLabel" name="skill2_traits_title">
          <property name="geometry">
           <rect>
            <x>290</x>
            <y>460</y>
            <width>121</width>
            <height>20</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {
	color: #FFFFFF;
	font: 12pt &quot;Century Gothic&quot;;
}</string>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Habilidad 2&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
         <widget class="QLabel" name="skill2_title">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>310</y>
            <width>151</width>
            <height>20</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {
	color: #FFFFFF;
	font: 16pt &quot;Century Gothic&quot;;
	width: 2px;
}</string>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Habilidad 2&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_4">
         <property name="minimumSize">
          <size>
           <width>468</width>
           <height>600</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
